tests/switchless_one_tcs/switchless_one_tcs.edl

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    enum num_tcs_t {
+        NUM_TCS = 1
+    };
+
+	enum status_e {
+        FUNC_NOT_START,
+        FUNC_WORKING,
+        FUNC_EXIT
+    };
+
+    trusted {
+        public void enc_empty_regular (void);
+        public void enc_empty_switchless(
+            [user_check] status_e *status ) transition_using_threads;
+    };
+
+    untrusted {
```

Comment:
If there are no ocalls I don't think you need this section

Reply:
sure, thanks